.sanitizers/undefined.supp

```diff
@@ -0,0 +1 @@
+unsigned-integer-overflow:_philox4x32bumpkey(r123array2x32)
```

Comment:
curious about this - is this something work reporting upstream or false-positive?

Reply:
It's not actually undefined behaviour, see https://clang.llvm.org/docs/UndefinedBehaviorSanitizer.html
> `-fsanitize=unsigned-integer-overflow`: Unsigned integer overflow, where the result of an unsigned integer computation cannot be represented in its type. Unlike signed integer overflow, this is not undefined behavior, but it is often unintentional. This sanitizer does not check for lossy implicit conversions performed before such a computation (see `-fsanitize=implicit-conversion`).

So this isn't a big red flag. I haven't yet investigated what's going on in the cases caught by our test suite; I thought it was better to get these checks in and then incrementally try to remove the suppressions in other, smaller PRs.

Reply:
I disabled this check in the NEURON PR because it was too noisy, BTW: https://github.com/neuronsimulator/nrn/blob/e347a9dfdcb475c63fc8a34ed7e058f3c2600593/CMakeLists.txt#L250